core/frontend/src/test/public/assets/wmts_capabilities/USGSHydroCached_capabilities.xml

```diff
@@ -0,0 +1,490 @@
+          <MatrixWidth>131076</MatrixWidth> 
+            <MatrixHeight>164449</MatrixHeight>
+          </TileMatrix>
+          <TileMatrix>
+          <ows:Identifier>18</ows:Identifier>
+          <ScaleDenominator>2132.729584141936</ScaleDenominator>
+          <TopLeftCorner>-2.0037508342787E7 2.0037508342787E7</TopLeftCorner>  
+          <TileWidth>256</TileWidth> 
+          <TileHeight>256</TileHeight>
+          <MatrixWidth>262152</MatrixWidth> 
+            <MatrixHeight>328898</MatrixHeight>
+          </TileMatrix>
+          <TileMatrix>
+          <ows:Identifier>19</ows:Identifier>
+          <ScaleDenominator>1066.3647915984968</ScaleDenominator>
+          <TopLeftCorner>-2.0037508342787E7 2.0037508342787E7</TopLeftCorner>  
+          <TileWidth>256</TileWidth> 
+          <TileHeight>256</TileHeight>
+          <MatrixWidth>524304</MatrixWidth> 
+            <MatrixHeight>657796</MatrixHeight>
+          </TileMatrix>
+          <TileMatrix>
+          <ows:Identifier>20</ows:Identifier>
+          <ScaleDenominator>533.1823957992484</ScaleDenominator>
+          <TopLeftCorner>-2.0037508342787E7 2.0037508342787E7</TopLeftCorner>  
+          <TileWidth>256</TileWidth> 
+          <TileHeight>256</TileHeight>
+          <MatrixWidth>1048608</MatrixWidth> 
+            <MatrixHeight>1315592</MatrixHeight>
+          </TileMatrix>
+          <TileMatrix>
+          <ows:Identifier>21</ows:Identifier>
+          <ScaleDenominator>266.5911978996242</ScaleDenominator>
+          <TopLeftCorner>-2.0037508342787E7 2.0037508342787E7</TopLeftCorner>  
+          <TileWidth>256</TileWidth> 
+          <TileHeight>256</TileHeight>
+          <MatrixWidth>2097216</MatrixWidth> 
+            <MatrixHeight>2631184</MatrixHeight>
+          </TileMatrix>
+          <TileMatrix>
+          <ows:Identifier>22</ows:Identifier>
+          <ScaleDenominator>133.2955989498121</ScaleDenominator>
+          <TopLeftCorner>-2.0037508342787E7 2.0037508342787E7</TopLeftCorner>  
+          <TileWidth>256</TileWidth> 
+          <TileHeight>256</TileHeight>
+          <MatrixWidth>4194432</MatrixWidth> 
+            <MatrixHeight>5262368</MatrixHeight>
+          </TileMatrix>
+          <TileMatrix>
+          <ows:Identifier>23</ows:Identifier>
+          <ScaleDenominator>66.64779947490605</ScaleDenominator>
+          <TopLeftCorner>-2.0037508342787E7 2.0037508342787E7</TopLeftCorner>  
+          <TileWidth>256</TileWidth> 
+          <TileHeight>256</TileHeight>
+          <MatrixWidth>8388863</MatrixWidth> 
+            <MatrixHeight>10524735</MatrixHeight>
+          </TileMatrix>
+          </TileMatrixSet>
+   <TileMatrixSet>
+      <ows:Title>GoogleMapsCompatible</ows:Title>
+      <ows:Abstract>the wellknown 'GoogleMapsCompatible' tile matrix set defined by OGC WMTS specification</ows:Abstract>
+      <ows:Identifier>GoogleMapsCompatible</ows:Identifier>
+      <ows:SupportedCRS>urn:ogc:def:crs:EPSG:6.18.3:3857</ows:SupportedCRS>
```

Comment:
@mdastous-bentley How are you going to convert such a CRS into something you can interpret as WebMercator? Previous entry in same capabilities used: "urn:ogc:def:crs:EPSG::3857" ... Is there a pattern and will some service be required to convert these identifiers into something meaningful? Also noticed the "CRS:84" was still part of WMTS specs while this pattern was ambiguous in WMS. Interpretation service can be either through the backend or through the GeoCoord service. Which would be most suitable in your architecture?

Reply:
In the parser I simply keep the CRS values as string.  Those values are URI than can be looked up using a web service like this:
http://www.opengis.net/def/crs/OGC/1.3/CRS84

I guess the Geoservice could be used to convert those definitions into a proper GCS definition.